packages/siwe/lib/client.ts

```diff
@@ -404,23 +404,21 @@ export class SiweMessage {
 
     const ISO8601 =
```

Comment:
You can remove this now

Reply:
How can I make sure that the format is ISO 8601 without it? Remember that the abnf parser only runs once when the message is deserialized.

Reply:
I just meant that this variable isn't used anymore, unless I'm missing something?

Reply:
I exec this on the function to check the date

Reply:
I'm confused, `eslint` agrees with me 
```
  347:13  warning  'ISO8601' is assigned a value but never used     @typescript-eslint/no-unused-vars
```
Not the only warning btw, it should be added to the CI

Reply:
I confused it with the utils.ts one 🤦